.gitignore

```
@@ -1,4 +1,6 @@
-./src
+src
+cmd/src/src
+cmd/src/*.zip
```

```suggestion
./src
./cmd/src/src
*.zip
```

I don't think you want a bare `src` here otherwise any file in `cmd/src` will be ignored from here on out.